### PR TITLE
Syntax of snapcraft.yaml (1.x)

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

**Comment by elopio** [comment]
You have a weird diff in here. I'm not sure how to help, I would just copy the .md files to /tmp, delete everything and try again :)

**Comment by kyrofa** [comment]
@elopio they look okay to me... which diff is weird?

**Comment by elopio** [comment]
@kyrofa on files changed I see examples/tomcat-maven-webapp/snapcraft.yaml, integration-tests/units/jobs.pxu, snapcraft/cmds.py, ...
Things that come from other PRs.

**Comment by kyrofa** [comment]
Ah, yeah this looks like it needs a rebase.

**Comment by dholbach** [comment]
Ok, I think I managed to do the rebase. O:-)

File: docs/snapcraft-syntax.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+What follows is a list of all the attributes the `snapcraft.yaml` file can
+contain.
+
+ * `name` (string)  
+   The name of the resulting snap.
+ * `version` (string)  
+   The version of the resulting snap.
+ * `vendor` (string)  
+   Vendor in email format as defined in RFC 5322.
```
Comment (kyrofa) [inline docs/snapcraft-syntax.md]: Would it be worth mentioning that this is deprecated here?

File: docs/snapcraft-syntax.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ * `summary` (string)  
+   A 78 character long summary for the snap.
+ * `description` (string)  
+   The description for the snap, this can and is expected to be a longer
+   explanation for the snap.
+ * `config` (string)  
+   Path to the runnable in snap that will be used as the Snappy config
+   interface.
```
Comment (kyrofa) [inline docs/snapcraft-syntax.md]: I personally (a relative newcomer to snappy) am not familiar with this. Have I missed a document, or should this section contain a reference link?

Comment (dholbach) [inline docs/snapcraft-syntax.md]: I'll add a link to https://developer.ubuntu.com/en/snappy/guides/config-command/

File: docs/snapcraft-syntax.md (hunk 1 above)
Comment (dholbach) [inline docs/snapcraft-syntax.md]: I think at some stage we'll just phase out the 1.x docs(?). With snapcraft 0.6 it's still required, right?

Comment (kyrofa) [inline docs/snapcraft-syntax.md]: Yeah, okay that makes sense.

**Comment by kyrofa** [comment]
I'm good with this. @dholbach would you mind squashing?

**Comment by dholbach** [comment]
Done.

**Comment by kyrofa** [comment]
@elopio, @sergiusens I'd like to get an OK from at least one of you before this merges.

File: docs/snapcraft-syntax.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+   interface](https://developer.ubuntu.com/snappy/guides/config-command/).
+ * `services` (yaml subsection)  
+   A set of keys representing service names with values as defined by the
+   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
+ * `binaries` (yaml subsection)  
+   A set of keys representing binary names with values as defined by the
+   [Snappy packaging spec](https://developer.ubuntu.com/snappy/guides/packaging-format-apps/).
+ * `icon` (string)  
+   Path to the icon that will be used for the snap.
+ * `license` (yaml subsection)
```
Comment (sergiusens) [inline docs/snapcraft-syntax.md]: this isn't actually implemented so I'd just remove it.

**Pull request merged**